src/property.html

```diff
@@ -0,0 +1,1066 @@
+                        Read More
+                      </a>
+                    </p>
+                  </div>
+                </div>
+                <!-- pricing -->
+                <div class="flex gap-4 items-center sm:gap-0 justify-between">
+                  <div>
+                    <p class="font-medium text-sm text-color-addition-info">
+                      Price
+                    </p>
+                    <p class="font-semibold text-sm sm:text-lg text-color">
+                      $650,000
+                    </p>
+                  </div>
+                  <div
+                    class="px-5 py-3.5 rounded-lg bg-for-second-btn font-medium text-xs sm:text-sm text-color xl:w-60 text-center"
+                  >
+                    View Property Details
+                  </div>
+                </div>
+              </div>
+              <div
+                class="p-5 sm:p-8 2xl:p-10 border-button rounded-xl gap-6 2xl:gap-8 xl:grid hidden"
+              >
+                <img
+                  src="/public/assets/propertypage/hotel.png"
+                  alt="hotel"
+                  class="w-full"
+                />
+                <!-- location hotel -->
+                <div class="flex flex-col gap-4 2xl:gap-5 items-start">
+                  <div
+                    class="text-xs sm:text-sm 2xl:text-lg font-medium text-color border-button bg-for-logo-section px-3 py-1.5 rounded-3xl"
+                  >
+                    Countryside Charm - Escape to Nature's Embrace
+                  </div>
+                  <!-- hotel rating and info -->
+                  <div class="flex flex-col gap-1">
+                    <p class="font-semibold text-lg xl:text-xl text-color">
+                      Rustic Retreat Cottage
+                    </p>
+                    <p
+                      class="font-medium text-sm xl:text-base text-color-addition-info"
+                    >
+                      Find tranquility in the countryside. This charming cottage
+                      is nestled amidst rolling hills...
+                      <a
+                        href="#"
+                        class="text-color text-underline-color-for-pricing-read-more"
+                      >
+                        Read More
+                      </a>
+                    </p>
+                  </div>
+                </div>
+                <!-- pricing -->
+                <div class="flex gap-4 items-center sm:gap-0 justify-between">
+                  <div>
+                    <p class="font-medium text-sm text-color-addition-info">
+                      Price
+                    </p>
+                    <p class="font-semibold text-sm sm:text-lg text-color">
+                      $350,000
+                    </p>
+                  </div>
+                  <div
+                    class="px-5 py-[14px] rounded-lg bg-for-second-btn font-medium text-[10px] sm:text-sm text-color xl:w-60 text-center"
```

Comment:
i told you earlier to put 14px padding to tailwind config. Why you do not listen to my advices. Plus we can add txt-[10px] to config file as well. why you did not do that?

Reply:
@hassnainzia121 ?

Reply:
fixed bro